examples/processed/ipoib-rdma-shared/10-nicclusterpolicy.yaml

```diff
@@ -33,6 +33,8 @@ spec:
             "rdmaHcaMax": 63,
             "selectors": {
               "ifNames": ["ibs2f0"]
+            }
+           }
```

Comment:
https://github.com/Mellanox/network-operator-docs/blob/main/examples/templates/ipoib-rdma-shared/10-nicclusterpolicy.yaml#L37please update the template